src/Atoms/HomeElements/DarkMode.test.js

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react';

```

Comment:
You are importing `screen`, but you are not using it. I highly recommend you to use screen, as we show it in the Workshop